test/tests/functional/pbs_advance_resv_end_hook.py

```diff
@@ -0,0 +1,169 @@
+from tests.functional import *
```

Comment:
File header is missing in PTL scripts.
Also, why do you have multiple test scripts. You are testing resv_end hook event, you can just create one test suite that will test the whole feature.